src/modrover/strategies/backward_explore.py

```diff
@@ -6,7 +6,7 @@ class BackwardExplore(RoverStrategy):
 
     def __init__(self, num_covariates: int):
         super().__init__(num_covariates)
-        self.base_learnerid = LearnerID(tuple(range(num_covariates + 1)))
+        self.base_learner_id = LearnerID(tuple(range(num_covariates + 1)))
```

Comment:
probably don't need the LearnerID constructor anymore right?

Reply:
Yes! that is a good point! Will change this